examples/xgboost/cyclic/config/config_fed_client.json

```diff
@@ -0,0 +1,32 @@
+{
+  "format_version": 2,
+
+  "executors": [
+    {
+      "tasks": [
+        "train"
+      ],
+      "executor": {
+        "id": "Executor",
+        "name": "LearnerExecutor",
+        "args": {
+          "learner_id": "xgboost-learner"
+        }
+      }
+    }
+  ],
+
+  "task_result_filters": [],
+  "task_data_filters": [],
+
+  "components": [
+    {
+      "id": "xgboost-learner",
+      "path": "xgboost_higgs_learner.XGBoostHiggsLearner",
+      "args": {
+        "train_config_filename": "config_train_uniform.json",
```

Comment:
Suggest adding a section in README to explain that this file can be changed.
(can be generated using your "prepare_train_config.py")

Reply:
added